akriti/tests/acceptance/features/createFile.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Create a file
+As a user
+I want to create new file
+So that I can store my data
+
+
+  Scenario: creates a new text file
+    Given the user has browsed to the login page
+    And the user has logged in with username "admin123" and password "12345" using webUI
```

Comment:
make admin id and password as default otherwise It will came problem in CI
```suggestion
    And the user has logged in with username "admin" and password "admin" using webUI
```